recipes-sota/rvi-sota-client/rvi-sota-client_git.bb

```diff
@@ -12,6 +12,9 @@ ${sysconfdir}/sota_client.version \
 ${sysconfdir}/sota_certificates \
 ${@bb.utils.contains('DISTRO_FEATURES', 'systemd', '${systemd_unitdir}/system/sota-client.service', '', d)} \
 ${@bb.utils.contains('DISTRO_FEATURES', 'systemd', '${systemd_unitdir}/system/sota-client-autoprovision.service', '', d)} \
+/var/sota/autoprov.url \
+/var/sota/sota_provisioning_url.env \
+/var/sota/sota_provisioning_credentials.p12 \
```

Comment:
This doesn't actually work, because of the way ostree treats /var. When you add things to /var, you have to do it in image_types_ostree.bbclass.